src/sst/core/config.cc

```diff
@@ -416,6 +416,8 @@ Config::insertOptions()
         "will be appended to the model options (or used as the model options if --model-options was not specified).",
         model_options_, false, false, true);
     DEF_FLAG_OPTVAL("print-timing-info", 0, "Print SST timing information", print_timing_, true);
+    DEF_ARG(
+        "timing-info-json", 0, "FILE", "Write SST timing information in JSON format", timing_json_, true, true, false);
```

Comment:
We had this option as not carrying over to a restart (second annotation).  I'm okay if we change that, but the --print-timing-info and --timing-info-json should match for whichever way we choose.